### PR TITLE
news item for announcements

### PR DESCRIPTION
**Pull request by jracusin**
This is the news item to go with the first announcement on announcements, code of conduct, and revisions.

### CODE REVIEW

File: app/routes/_gcn/route.tsx [hunk 1]
```diff
@@ -29,10 +29,10 @@ function Document({ children }: { children?: ReactNode }) {
     <>
       <Header />
       <NewsBanner>
-        New Swift-BAT/GUANO and IceCube Notice Types Available! See{' '}
+        New Announcement Feature, Code of Conduct, Circular Revisions. See{' '}
         <Link
           className="usa-link"
-          to="/news#new-swift-bat/guano-and-icecube-notices-available-via-kafka"
+          to="/news##new-announcement-feature-code-of-conduct-circular-revisions"
```
Comment (dakota002) [inline app/routes/_gcn/route.tsx:35]:
```suggestion
          to="/news#new-announcement-feature-code-of-conduct-circular-revisions"
```

**Comment by dakota002** [comment]
There is an extra `#` in the link, but I think it looks good otherwise

File: app/routes/_gcn.news._index.mdx [hunk 2]
```diff
@@ -24,6 +24,27 @@ import { Anchor } from '~/components/Anchor'
...
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'February 28, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>New Announcement Feature, Code of Conduct, Circular Revisions</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        - **Announcements from the GCN Team to our User Community:**  These announcements will contain information on new GCN features, changes to GCN Notices schema, new GCN Notices topics, and planned system outages. All GCN users are automatically subscribed to GCN Announcements.  You can modify your subscription at any time by following the personalized link at the bottom of announcement emails, or logging into your account, selecting "Email Notifications" in the user dropdown menu and toggling the announcements on/off.
```
Comment (lpsinger) [inline app/routes/_gcn.news._index.mdx:39]:
```suggestion
        - **Announcements from the GCN Team to our User Community:**  These announcements will contain information on new GCN features, changes to GCN Notices schema, new GCN Notices topics, and planned system maintenance. All GCN users are automatically subscribed to GCN Announcements.  You can modify your subscription at any time by following the personalized link at the bottom of announcement emails, or logging into your account, selecting "Email Notifications" in the user dropdown menu and toggling the announcements on/off.
```

File: app/routes/_gcn.news._index.mdx [hunk 3]
```diff
@@ -24,6 +24,27 @@ import { Anchor } from '~/components/Anchor'
...
+
+        - The [**GCN Code of Conduct**](https://gcn.nasa.gov/docs/code-of-conduct) describes policies for community behavior as well as mechanisms for retractions, errata, reproduction of GCN Notices and Circulars, enforcement, and acknowledgements.  These policies represent the informal standards that have supported the thriving GCN Community over the last 32 years, but benefits from formal description.
```
Comment (lpsinger) [inline app/routes/_gcn.news._index.mdx:41]:
```suggestion
        - The [**GCN Code of Conduct**](https://gcn.nasa.gov/docs/code-of-conduct) describes policies for community behavior as well as mechanisms for retractions, errata, reproduction of GCN Notices and Circulars, enforcement, and acknowledgements.  These policies represent the informal standards that have supported the thriving GCN Community over the last 32 years, but benefit from formal description.
```

If they are informal, then how can they have enforcement mechanisms?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/_gcn.news._index.mdx (hunk 3 above)
Comment (jracusin) [reply app/routes/_gcn.news._index.mdx]: I suppose they formalize the informal standards of the past

**Review by lpsinger — APPROVED** [review]

**Comment by lpsinger** [comment]
Looks good! I've approved it but marked it as a draft so that we don't accidentally merge it prematurely.

**Comment by jracusin** [comment]
When we are ready to merge, we need to update the date on the news & announcement page.

**Comment by dakota002** [comment]
> When we are ready to merge, we need to update the date on the news & announcement page.

Can you include that in this PR?

**Pull request merged**